include/alpaka/rand/RandStl.hpp

```diff
@@ -69,15 +90,87 @@ namespace alpaka
                     std::mt19937 m_State;
                 };
 
+                //#############################################################################
+                //! "Tiny" state mersenne twister implementation
+                //!
+                //! repository: github.com/MersenneTwister-Lab/TinyMT
+                //!
+                //! @license 3-clause BSD
+                //!
+                //! @author Mutsuo Saito (Hiroshima University)Tokio University.
+                //! @author Makoto Matsumoto (The University of Tokyo)
+                //!
+                //! size of state: 28 bytes (127 bits?!)
```

Comment:
I don't understand where [this](https://github.com/MersenneTwister-Lab/TinyMT/blob/master/tinymt/tinymt32.h#L33-L41) is 127 bits. At least for us its 28 bytes (228 bits) as we use it.